saleor/graphql/order/tests/mutations/test_draft_order_update.py

```diff
@@ -1163,3 +1165,166 @@ def test_draft_order_update_no_shipping_method_channel_listings(
+            ) {
+                errors {
+                    field
+                    code
+                    message
+                }
+                order {
+                    discounts {
+                        amount {
+                            amount
+                        }
+                        valueType
+                        type
+                        reason
+                    }
+                    lines {
+                        quantity
+                        unitDiscount {
+                          amount
+                        }
+                        undiscountedUnitPrice {
+                            net {
+                                amount
+                            }
+                        }
+                        unitPrice {
+                            net {
+                                amount
+                            }
+                        }
+                        totalPrice {
+                            net {
+                                amount
+                            }
+                        }
+                        unitDiscountReason
+                        unitDiscountType
+                        unitDiscountValue
+                        isGift
+                    }
+                    total {
+                        net {
+                            amount
+                        }
+                    }
+                    subtotal {
+                        net {
+                            amount
+                        }
+                    }
+                    undiscountedTotal {
+                        net {
+                            amount
+                        }
+                    }
+                }
+            }
+        }
+        """
+
+
+def test_draft_order_update_order_promotion(
+    staff_api_client,
+    permission_group_manage_orders,
+    customer_user,
+    shipping_method,
+    graphql_address_data,
+    variant_with_many_stocks,
+    channel_USD,
+    draft_order,
+    order_promotion_rule,
+):
+    # given
+    query = DRAFT_ORDER_UPDATE_MUTATION_PROMOTIONS
+    permission_group_manage_orders.user_set.add(staff_api_client.user)
+
+    rule = order_promotion_rule
+    promotion_id = graphene.Node.to_global_id("Promotion", rule.promotion_id)
+    assert rule.reward_value_type == RewardValueType.PERCENTAGE
+    assert rule.reward_value == Decimal("25")
+
+    variables = {
+        "id": graphene.Node.to_global_id("Order", draft_order.pk),
+        "input": {
+            "billingAddress": graphql_address_data,
+        },
+    }
+
+    # when
+    response = staff_api_client.post_graphql(query, variables)
+
+    # then
+    content = get_graphql_content(response)
+    assert not content["data"]["draftOrderUpdate"]["errors"]
+    order = content["data"]["draftOrderUpdate"]["order"]
+
+    assert len(order["discounts"]) == 1
+    assert order["discounts"][0]["amount"]["amount"] == 17.50
```

Comment:
Where did this value come from? It should taken from some input variable/object or calculated